config/src/main/java/org/hyperledger/besu/config/experimental/ExperimentalEIPs.java

```diff
@@ -24,21 +24,9 @@
   // To make it easier for tests to reset the value to default
   public static final long EIP1559_BASEFEE_DEFAULT_VALUE = 1000000000L;
```

Comment:
What do you think about deleting this class entirely and moving these constants? There are no `ExperimentalEIPs` anymore.